src/core/inlines.js

```diff
@@ -50,10 +50,10 @@ function inlineRefMatches(matched) {
   // slices "[[[" at the beginning and "]]]" at the end
   const ref = matched.slice(3, -3).trim();
   if (!ref.startsWith("#")) {
-    return hyperHTML`<a data-cite="${ref}"></a>`;
+    return hyperHTML`<a class="respec-inline-expansion" data-cite="${ref}"></a>`;
```

Comment:
Is this only for testing?

Reply:
I was thinking of styling these... I was going to add the "[[[" "]]]" back in using CSS generated content.

Reply:
(styled when they fail, that is)

Reply:
@saschanaz, decided to ditch the "respec-inline-expansion" class.